webpack/assets/javascripts/react_app/components/AutoComplete/AutoCompleteActions.js

```diff
@@ -0,0 +1,75 @@
+import URI from 'urijs';
+import API from '../../API';
+import { STATUS } from '../../constants';
+import {
+  AUTO_COMPLETE_REQUEST,
+  AUTO_COMPLETE_SUCCESS,
+  AUTO_COMPLETE_FAILURE,
+  AUTO_COMPLETE_RESET,
+  TRIGGERS,
+} from './AutoCompleteConstants';
+import { objectDeepTrim } from '../../common/helpers';
+
+export const getResults =
```

Comment:
If you want to practice `async/await` syntax, I think it will make this method looks better :)